### ISSUE TITLE
[Bug] False negative when apostrophe and nested tags are inside a tag

### ISSUE THREAD

**Issue by bdbernardy** [malinindev/check-unused-css#22]
I found an edge case that is causing the `check-unused-css` tool to fail. I managed to narrow down the issue to the simple test case:
an outer `div` tag that includes a text node with an apostrophe, and another node with a css class.

I'll try to fix it myself if you don't have time.

Thank you for the great work! Your tool is fantastic.

**WithApostropheInText.module.css**
```css
.usedClass1 {
  display: none;
}

.usedClass2 {
  display: none;
}
```

**WithApostropheInText.tsx**
```ts
export function WithApostropheInText() {
  return (
    <div
      className={
        styles.usedClass1
      }
    >
      The ' character is making
      <span
        className={styles.usedClass2}
      >
        the test fail
      </span>
    </div>
  );
}
```

**Comment by malinindev** [comment]
Thanks for the report, glad the tool is helping!

I reproduced the bug using your test case. It's a real edge case - the `'` inside jsx text breaks the parser. I originally avoided using an AST for simplicity, but the logic keeps growing

Interestingly, `decomment` has the same problem - everything after an apostrophe in JSX gets stripped

I'll fix and publish an update soon

**Comment by malinindev** [comment]
Fixed in v0.1.7. Replaced regexp parser with AST. Slightly slower, but more robust.
Let me know if you run into anything - issues and PRs are welcome

**Comment by bdbernardy** [comment]
> Fixed in v0.1.7. Replaced regexp parser with AST. Slightly slower, but more robust. Let me know if you run into anything - issues and PRs are welcome

It worked like a charm. Thanks for your help!
